.github/pull_request_template.md

```diff
@@ -1,26 +1,17 @@
-## Pull Request Info
+# Pull Request Info
 
-[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
-
-### Issue JIRA link:
-https://jira.mongodb.org/browse/DOCSP-NNNNN
+[JIRA](https://jira.mongodb.org)
+[Staging](https://www.example.com)
 
-### Snooty build log:
-**Paste your workerpool job link here**
+## Reviewing
 
-### Docs staging link (requires sign-in on MongoDB Corp SSO):
-https://docs-mongodborg-staging.corp.mongodb.com/java/docsworker-xlarge/NNNNNNNNNNN/
+[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
 
-### Self-Review Checklist
+## Self-Review Checklist
 
 - [ ] Is this free of any warnings or errors in the RST?
 - [ ] Did you run a spell-check?
-- [ ] Did you run a grammar-check?
-- [ ] Does it render on staging correctly?
+- [ ] Did you run a grammar check?
 - [ ] Are all the links working?
-- [ ] Are the staging and workerpool job links in the PR description updated?
-
-### If your page documents a concept, does it meet the following criteria?
-
-- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
-- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)
+- [ ] (Employees) Does it render on staging correctly?
+- [ ] (Employees) Is the build log free of errors?
```

Comment:
Suggestion: Remove "(Employees)".
We decided we aren't going to recommend community contributions, so these don't have to be in a form that the community understands.